src/index.html

```diff
@@ -7,6 +7,28 @@
     <title>Leadboard</title>
 </head>
 <body>
-    <h1 >Leadboard</h1>
+    <h1 class="p-4">Leadboard</h1>
+    <main class="d-flex container justify-content-around">
+        <section class="score">
+            <label for="refresh">Recent scores</label>
+            <button id="refresh">Refresh</button>
+            <div class="scores border border-3 border-dark m-4">
+                <h5 class="p-2">Name : 100</h5>
+                <h5 class="bg-success p-2 text-dark bg-opacity-10">Name : 100</h5>
+                <h5 class="p-2">Name : 100</h5>
+                <h5 class="bg-success p-2 text-dark bg-opacity-10">Name : 100</h5>
+                <h5 class="p-2">Name : 100</h5>
+                <h5 class="bg-success p-2 text-dark bg-opacity-10">Name : 100</h5>
+            </div>
+        </section>
+        <section class="add-score">
+            <h3>Add your score</h3>
+            <form action="#" class="d-flex flex-column">
+                <input type="text" placeholder="Your name" class="m-3">
+                <input type="text" placeholder="Your score" class="m-3">
+                <button class="m-3">Submit</button>
+            </form>
+        </section>
+    </main>
```

Comment:
- [ ] Please make your design like the wireframe as you should make the scores table wider and the submit button to the right.

Your Design | Wireframe Design
------------- | --------------------
![Capture3](https://user-images.githubusercontent.com/33752735/187730182-a8660b4e-3468-4fb0-be1a-e7e9931f39be.PNG) | ![leaderboard_wireframe](https://user-images.githubusercontent.com/33752735/187730212-c44c33fc-d249-4cb2-9ff6-da08b82c88a4.png)

Reply:
Thank you for your suggestion.
Changed the same.